### DISCUSSION TITLE
Some searches end with a 503 error

### DISCUSSION THREAD

**Discussion by Tealk** [mastodon/mastodon#18450] [Q&A] [answered]
Hi,

when I search the name below on my instance I get a 503, is this now due to my Mastodon configuration?
```
XHRGEThttps://rollenspiel.social/api/v2/search?q=@schnupperrunde@rollenspiel.events&resolve=true&limit=5
[HTTP/2 503 Service Unavailable 33ms]
```

My mastodon is behind a nginx proxy manager, is there anything special I need to be aware of?

**Comment by ClearlyClaire** [comment]
The exact cause of a 503 error can vary, but it is generally a network error when trying to fetch the remote account, or the remote server erroring out. Are you still experiencing this? Does it occur with other instances? It seems your network setup may prevent the two servers from connecting to each other.

**Reply by Tealk** [reply]
Yes the problem is that both servers are reachable under the same IP and that is probably not covered by IP.

**Comment by Tealk** [comment, marked as answer]
Now I have set my vLAN so that the server directly address each other, but now get the message:

> 422 Mastodon::HostValidationError on https://rollenspiel.events/.well-known/webfinger?resource=acct:schnupperrunde@rollenspiel.events

**Reply by ClearlyClaire** [reply]
This is a security measure to prevent confused deputy attacks, see https://docs.joinmastodon.org/admin/config/#allowed-private-addresses if you want to explicitly allow requests to your local networks

**Reply by Tealk** [reply]
Thank you very much, I have completely overlooked this
